otelcolbuilder/cmd/collector_windows.go

```diff
@@ -0,0 +1,247 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build windows
+
+// This is a copy of https://github.com/open-telemetry/opentelemetry-collector/blob/20710ff2aeaed3fe1c3e46423291522674db44d6/otelcol/collector_windows.go.
+// This is required to maintain our command line flags when running the collector as a Windows service, until either
+// the necessary names become public, or there's a better way of customizing config providers.
```

Comment:
it looks like `NewSvcHandler` is public already, which seems to be the only thing we use in the patch, do we need other names/functions to be public? If so, maybe we could identify those and create an upstream issue for it.